src/ending/ceremony_and_credits.c

```diff
@@ -566,191 +564,192 @@ void init_cinematic_camera(void) {
         vec3s_clear(D_80287750[i].unkC);
     }
 
-    D_802856C0 = 0.0f;
+    gSizeSlidingBorders = 0.0f;
 
     if (gGamestate == ENDING) {
         D_802856B0 = 120.0f;
         D_802856B4 = 12.0f;
-        D_802856B8 = 120.0f;
+        gOrderedSizeSlidingBorders = 120.0f;
     } else {
         D_802856B0 = 98.0f;
         D_802856B4 = 12.0f;
-        D_802856B8 = 52.0f;
+        gOrderedSizeSlidingBorders = 52.0f;
     }
 }
 
+// weird link between camera and cinematic camera where pos and lookAt are swapped
 s32 func_80283648(Camera *camera) {
-    s16 sp6E;
-    s16 sp6C;
+    s16 angleYToXZ;
+    s16 angleY;
     f32 var_f2;
-    f32 sp64;
-    Vec3f sp58;
-    Vec3f sp4C;
-    Vec3f sp40;
-    struct CinematicCamera *new_var = &D_802876E0;
-
-    func_80283428();
-    vec3f_copy_return_dupe(sp58, camera->pos);
-    vec3f_copy_return_dupe(sp4C, camera->lookAt);
-    vec3f_copy_return_dupe(sp40, camera->up);
-    new_var->cutscene = func_8028336C(new_var, camera);
-    if (new_var->cutscene != 0) {
-        vec3f_copy_return_dupe(new_var->lookAt, camera->pos);
-        vec3f_copy_return_dupe(new_var->pos, camera->lookAt);
-        play_cutscene(new_var);
-        func_80282454(new_var->lookAt, new_var->pos, &sp64, &sp6E, &sp6C);
-        if (sp6E >= 0x3800) {
-            sp6E = 0x3800;
+    f32 distance;
+    Vec3f pos;
+    Vec3f lookAt;
+    Vec3f up;
+    CinematicCamera *cinematicCamera = &D_802876E0;
+
+    cinematic_empty_0_function();
```

Comment:
`cinematic_useless_function`?

Reply:
cinematic_stub()

Reply:
it's just a function who return 0 and are use, so useless no, and I don't think stub are good also

Reply:
Despite that it's used. It doesn't actually do anything with the return value (At least not that I could see).

I think _stub is suitable to show that there's not really anything interesting here.